libs/filamat/src/shaders/CodeGenerator.cpp

```diff
@@ -509,6 +509,8 @@ char const* CodeGenerator::getConstantName(MaterialBuilder::Property property) n
         case Property::SUBSURFACE_POWER:     return "SUBSURFACE_POWER";
         case Property::SUBSURFACE_COLOR:     return "SUBSURFACE_COLOR";
         case Property::SHEEN_COLOR:          return "SHEEN_COLOR";
+        case Property::GLOSSINESS:          return "GLOSSINESS";
```

Comment:
alignment of "return" on both lines. Tabs?

Reply:
Fixed